<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>346</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>246</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>246</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: rgb(63, 63, 63);</string>
       </property>
       <property name="text">
        <string>Temp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineTemp">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(243, 106, 72);
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(63, 63, 63);</string>
       </property>
       <property name="text">
        <string>Feels Like</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineFeels">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(243, 106, 72);
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color: rgb(63, 63, 63);</string>
       </property>
       <property name="text">
        <string>Latitud</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineLatitud">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(243, 106, 72);
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">color: rgb(63, 63, 63);</string>
       </property>
       <property name="text">
        <string>Longitud</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineLongitud">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(243, 106, 72);
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">color: rgb(63, 63, 63);</string>
       </property>
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineIcon">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(243, 106, 72);
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">color: rgb(63, 63, 63);</string>
       </property>
       <property name="text">
        <string>Country</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineCountry">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(243, 106, 72);
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">color: rgb(63, 63, 63);</string>
       </property>
       <property name="text">
        <string>Place</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linePlace">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(243, 106, 72);
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">color: rgb(63, 63, 63);</string>
       </property>
       <property name="text">
        <string>Pressure</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linePressure">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(243, 106, 72);
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">color: rgb(63, 63, 63);</string>
       </property>
       <property name="text">
        <string>Humidity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineHumidity">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(243, 106, 72);
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
